adserver/analyzer/views.py

```diff
@@ -74,19 +73,9 @@ def get(self, request):
                 )
 
             return Response(
-                f"""
-                <h2>Results:</h2>
-                <ul>
-                    <li><a href="{urls[0].url}">{urls[0].url}</a></li>
-                    <li><a href="{urls[1].url}">{urls[1].url}</a></li>
-                    <li><a href="{urls[2].url}">{urls[2].url}</a></li>
-                    <li><a href="{urls[3].url}">{urls[3].url}</a></li>
-                </ul>
-                <h2>
-                Text:
-                </h2>
-                <textarea style="height:100%; width:80%" disabled>
-                {processed_text}
-                </textarea>
-                """
+                {
+                    "count": len(urls),
+                    "text": processed_text[:500],
+                    "results": [[url.url, url.distance] for url in urls],
```

Comment:
I would consider returning a dict here instead of a tuple as we'll want to return more things (description, title, etc.).

Reply:
Yea, makes sense.

Reply:
Key attributes to return are url, description, title, and we could just return the favicon as the image (append `favicon.ico` to the URL domain). If desired, you could skip the image and the frontend could do that.